psycop/common/model_training_v2/hyperparameter_suggester/test_hyperparameter_suggester.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
+
+import optuna
+from optuna.testing.storage import StorageSupplier
+
+from psycop.common.model_training_v2.hyperparameter_suggester.hyperparameter_suggester import (
+    SearchSpace,
+    hyperparameter_suggester,
+)
+from psycop.common.model_training_v2.hyperparameter_suggester.suggesters.logistic_regression_suggester import (
+    LogisticRegressionSuggester,
+)
+from psycop.common.model_training_v2.hyperparameter_suggester.suggesters.test_suggesters import (
+    float_space_for_test,
+)
+
+
+class TestHyperparameterSuggster:
+    def _get_suggestions(self, base_cfg: dict[str, Any]) -> dict[str, Any]:
+        sampler = optuna.samplers.RandomSampler()
+
+        with StorageSupplier("inmemory") as storage:
+            study = optuna.create_study(storage=storage, sampler=sampler)
+            trial = optuna.Trial(
+                study,
+                study._storage.create_new_trial(study._study_id),  # type: ignore
+            )
+
+            return hyperparameter_suggester(base_cfg=base_cfg, trial=trial)
+
+    def test_hyperparameter_suggester(self):
```

Comment:
consider parametrizing this so we can easily test multiple different suggesters. But looks good!